libs/IO/Image.h

```diff
@@ -107,6 +109,49 @@ class IO_API CImage
 	#endif
 
 protected:
+    template<typename T>
+    static void findMinMaxPercentile(void *data, Size size, T* min, T* max){
```

Comment:
pls use & or simply return a std::pair
and use the same formatting as used in the rest of the library

Reply:
Done :+1: